hrrecruitment/tests/phpunit/HRRecruitmentTestTrait.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * A trait with helper methods to be reused among this extension's tests
+ */
+trait HRRecruitmentTestTrait {
```

Comment:
I don't think we need this class at all. See my comments about the createContact() vs Contact::createIndividual() and about extending PHPUnit_Framework_TestCase